bundles/config/org.eclipse.smarthome.config.xml/src/main/java/org/eclipse/smarthome/config/xml/ConfigDescriptionConverter.java

```diff
@@ -42,7 +46,6 @@ public ConfigDescriptionConverter() {
         this.attributeMapValidator = new ConverterAttributeMapValidator(new String[][] { { "uri", "false" } });
     }
 
-    @SuppressWarnings("unchecked")
```

Comment:
Eclipse gives me a warning when it's there, and when I remove it, it doesn't... Does your environment show something different?